assembl/static2/js/app/pages/brightMirrorFiction.jsx

```diff
@@ -105,6 +105,13 @@ type BrightMirrorFictionState = {
   publicationState: string
 };
 
+type CommentsInfo = {
+  /** Top comments build from transformPosts function */
+  topComments: [Object],
```

Comment:
'Object' Could it be defined in details ?

Reply:
Since `transformPosts` is not described, I don't know the output format.